packages/express/lib/serve.js

```diff
@@ -47,6 +49,12 @@ module.exports = (mode, { configureServer }) => {
       }
     });
   });
+  // eslint-disable-next-line no-unused-vars
+  app.use((err, req, res, next) =>
+    finalhandler(req, res, {
```

Comment:
Express.js uses this module as its final catch-all and default error handler. In Express, it is configured to dump errors (including stack traces) directly into the console.

Since I want to leave it up to `untool` (or hops etc.) to control what to do about (recoverable) errors, I introduced a slightly modified config here.